docs/reference/setup/install/connect-clients.asciidoc

```diff
@@ -5,9 +5,11 @@ HTTP layer. A CA certificate is generated and stored on disk, and the hex-coded
 SHA-256 fingerprint of this certificate is also output to the terminal. Any 
 clients that connect to {es}, such as the 
 https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} Clients],
-{beats}, {ls}, and {fleet} must validate that they trust the certificate that
-{es} uses for HTTPS. Clients can establish trust by using either the fingerprint
-of the CA certificate or the CA certificate itself.
+{beats}, standalone {agent}s, and {ls} must validate that they trust the
+certificate that {es} uses for HTTPS. {fleet-server} and {fleet}-managed
+{agent}s are automatically configured to trust the fingerprint of the CA.
+Other clients can establish trust by using either the fingerprint of the CA
+certificate or the CA certificate itself.
 
 If the auto-configuration process already completed, you can still obtain the 
 fingerprint of the security certificate. You can also copy the CA certificate
```

Comment:
I know this is not part of this change but we can also mention the path to the CA certificate  in these instructions which is always : `$ES_HOME/config/certs/http_ca.crt` and later on in this file, maybe in the top after the " You can also copy the CA certificate to your machine and configure your client to use it. " section ? 

Also, we can remove the 
`<timestamp>:: The timestamp of when the auto-configuration process created the security files directory.` callout as it is a remnant of a previous iteration.

Reply:
@dedemorton Let me know how you want to handle this. I'm happy to do one of the following:

A) Perform this work in a separate PR once this is merged in.
B) Push these changes to your branch.
C) Let you make the changes on this branch.

I mostly don't want to block you or push stuff to your PR without your permission.